os/drivers/Makefile

```diff
@@ -75,6 +75,7 @@ include audio$(DELIM)Make.defs
 include bch$(DELIM)Make.defs
 include bluetooth$(DELIM)Make.defs
 include cpuload$(DELIM)Make.defs
+include compression$(DELIM)Make.defs
```

Comment:
Minor comment.
Let's location this before `cpuload` by Alphabet order.

Reply:
done